src/definitions/_types.ts

```diff
@@ -163,3 +163,11 @@ export type NexusGraphQLInterfaceTypeConfig = WithExt<
 export type NexusGraphQLSchema = Omit<GraphQLSchema, 'extensions'> & {
   extensions: { nexus: NexusSchemaExtension }
 }
+
+export interface NexusFeatures {
+  abstractTypes?: {
```

Comment:
```suggestion
  /**
   * Toggle abstract type implementation strategies. For more info please refer to to the [abstract types guide](https://nxs.li/guides/abstract-types)
   */
  abstractTypes?: {
```

We can do a lot more jsdoc work here and on each strategy as well. Bit hard to do that via code suggest though.

What I would say is that we can copy the content from each strategy section in the abstract types guide into jsdoc here.

Kind of intense but going to be pretty helpful I bet.